### PR TITLE
removes name based service discovery

### PR DESCRIPTION
**Pull request by kotharironak**
kube internally uses `name: http` property for service discovery, this was causing issue with `serviceSelectorLabels` via diverting traffic to federated service in helm setup. So, like any other service, removing it.

### CODE REVIEW

File: helm/templates/service.yaml [hunk 1]
```diff
@@ -8,8 +8,6 @@ spec:
   type: {{ .Values.service.type }}
   ports:
     - port: {{ .Values.service.port }}
-      targetPort: http
```
Comment (ravisingal) [inline helm/templates/service.yaml:11]: why remove `targetPort`? where will service forward request to?

Comment (kotharironak) [reply helm/templates/service.yaml:11]: reverting this back.

**Comment by kotharironak** [comment]
Hold on, this may not require, checking with @ravisingal we had an issue with `name:http` with istio + grpc server. here we are using jetty, so it won't be an issue. we are testing without this change for dev helm setup.

**Comment by kotharironak** [comment]
This is not required, closing this ticket. It was an issue of missing `name` based containerPort in federated service helm.

**Pull request closed without merging**